Adding a readme for MSCRNN in EdgeML repo

adityakusupati opened the pull request:
This PR is to link the codebase of MSC-RNN with the original EdgeML repo along with resources.

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/microsoft/EdgeML?pullRequest=143) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/EdgeML?pullRequest=143) before we can accept your contribution.<br/><br/>:x: adityakusupati [sign now](https://cla.opensource.microsoft.com/microsoft/EdgeML?pullRequest=143)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/microsoft/EdgeML?pullRequest=143) it.</sub>

harsha-simhadri commented:
> This PR is to link the codebase of MSC-RNN with the original EdgeML repo along with resources.

Can you add the PDF to the pubs folder, or would you prefer to keep it in arXiv?

Also,  should there a pointer in the main README?

adityakusupati commented:
I am fine with adding the PDF to the pubs folder if EdgeML is OK with it. 
I will add a pointer near GesturePod in the main README.

adityakusupati commented:
@harsha-simhadri please check it now. I have added the publication to the repo. Added links to ACM DL, arXiv as well. 
I have edited the main readme to talk about the MSC-RNN paper and added it along side Gesturepod.
Let me know if you want me to change anything else.
